cli_tools/gce_onestep_image_import/onestep_importer/aws_importer/args.go

```diff
@@ -0,0 +1,134 @@
+package awsimporter
```

Comment:
as said, use on single package for importer. No benefits for multiple packages here. Keep those classes private.